src/main/scala/temple/builder/OrchestrationBuilder.scala

```diff
@@ -36,7 +36,7 @@ object OrchestrationBuilder {
             dbLifecycleCommand = dbLanguage match {
               case Database.Postgres => LifecycleCommand.psqlSetup.toString
             },
-            usesAuth = true,
+            usesAuth = if (name == "auth") false else true,
```

Comment:
Gives me confidence for the next 1124 lines
```suggestion
            usesAuth = name != "auth"
```